src/app-common/components/common/movies-list/MoviesList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import {Link} from 'react-router-dom';
+import Card from "../card/Card.js";
+
+class MoviesList extends Component {
+
+    constructor(props){
+        super(props);
+    }
+
+    render() {
+        const {movies, getCurrentMovie, currentMoviesGenres, currentMovie} = this.props;
+        let obj;
+
+        return (
+            <div className="content__outer">
+                {
+                    currentMovie ? (this.obj = currentMoviesGenres) : (this.obj = movies),
```

Comment:
We don't need to use _this_ because we don't use component state. Our local variable _obj_ left unused. That logic would be better to put outside _return_:
`let obj = currentMovie ? currentMoviesGenres : movies;`